buildSrc/src/main/groovy/dwh-migration-dumper.java-common-conventions.gradle

```diff
@@ -57,7 +53,7 @@ dependencies {
         def guavaVersion =  '31.1-jre'
         implementation "com.google.guava:guava:$guavaVersion"
         testFixturesImplementation "com.google.guava:guava:$guavaVersion"
-        def jacksonVersion = '2.12.7'
+        def jacksonVersion = '2.15.0-rc1' // first version to use snakeyaml v2+ for vulnerability fix
```

Comment:
Do we have to force the new jackson, or can we use the old jackson, but force a newer snakeyaml?

Reply:
Done; I've forced snakeyaml to 2.0 instead.